src/components/TransactionItemRow/index.tsx

```diff
@@ -47,10 +61,61 @@ type TransactionWithOptionalSearchFields = TransactionWithOptionalHighlight & {
     /** The personal details of the user paying the request */
     to?: SearchPersonalDetails;
 
+    /** formatted "to" value used for displaying and sorting on Reports page */
```

Comment:
Can you capitalize all of these please?
```suggestion
    /** Formatted "to" value used for displaying and sorting on Reports page */
```